demo/src/screens/componentScreens/HintsScreen.tsx

```diff
@@ -215,6 +223,19 @@ export default class HintsScreen extends Component<HintScreenProps> {
               </View>
             </>
           )}
+
+          <View marginT-100 row center>
+            {targetPosition !== 'flex-start' && <Text marginH-s3>Text pushing button</Text>}
+            <Hint
+              message={'Hint'}
+              visible={showSecondHint}
+              onBackgroundPress={this.toggleSecondHint}
+              useSideTip={false}
```

Comment:
useSideTip must be false or we can play with this from the switches?

Reply:
Passing true will result in a wrong UI that looks bad, it's not a bug, it's just not meant for that so I rather pass false